doc/rrdinfo.pod

```diff
@@ -5,7 +5,8 @@ rrdinfo - extract header information from an RRD
 =head1 SYNOPSIS
 
 B<rrdtool> B<info> I<filename>
-S<[B<--daemon>|B<-d> I<address> S<[B<--noflush>|B<-F>]>]>
+
```

Comment:
This additional line would lead to:
<pre>
rrdtool info filename

[--daemon|-d address [--noflush|-F]]
<pre>